collector/consumer/exporter/otelexporter/defaultadapter/otel_adapter_cache.go

```diff
@@ -0,0 +1,349 @@
+package defaultadapter
+
+import (
+	"github.com/Kindling-project/kindling/collector/model"
+	"github.com/Kindling-project/kindling/collector/model/constlabels"
+	"github.com/Kindling-project/kindling/collector/model/constvalues"
+	"go.opentelemetry.io/otel/attribute"
+	"sort"
+	"strconv"
+)
+
+type adapterCache struct {
```

Comment:
Could you add a comment on this line to describe what `adapterCache` does and why it is named as "cache"?